### PR TITLE
Update Contribute Readme.md

### PR DESCRIPTION
**Pull request by pavlenex**
- Make it more concise
- Link to ContributeDevTest.md
- Re-phrase to make language more suitable for documentation and easier fot translations in the future.

### CODE REVIEW

File: docs/Contribute/README.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 
-Contributors can help in a number of ways:
-* [Coding for BTCPay Server](../Contribute/ContributeDev/README.md)
-* [Writing for BTCPay Server](../Contribute/ContributeWrite/README.md)
-* [Designing for BTCPay Server](ContributeDesign.md)
-* [Translating for BTCPay Server](ContributeTranslate.md)
+Depending on your skills and interest, you can help in a number of ways:
+
+* [Code and development](../Contribute/ContributeDev/README.md)
+*  [Testing](../ContributeDevTest.md)
```
Comment (Zaxounette) [inline docs/Contribute/README.md]:
```suggestion
* [Testing](../ContributeDevTest.md)
```

**Review by Zaxounette — CHANGES_REQUESTED** [review]
Just a superfluous `space`, otherwise it's a go for me.

File: docs/Contribute/README.md (hunk 1 above)
Comment (pavlenex) [reply docs/Contribute/README.md]: It's not, it should create a sub-bullet

Comment (Zaxounette) [reply docs/Contribute/README.md]: I'm pretty sure it won't.
Should be a `Tab` before the bullet point.

**Review by Zaxounette — APPROVED** [review]

**Pull request merged**